packages/react-storage/src/components/StorageBrowser/views/LocationActionView/DeleteFilesControls.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Controls } from '../Controls';
+import { ButtonElement } from '../../context/elements';
+import { DataTableControl } from '../../controls/DataTableControl';
+import { ControlsContextProvider } from '../../controls/context';
+import { CLASS_BASE } from '../constants';
+import { Title } from './Controls/Title';
+import { useDeleteActionView } from './hooks/useDeleteActionView';
+import { StatusDisplayControl } from '../../controls/StatusDisplayControl';
+import { ControlsContext } from '../../controls/types';
+
+const { Exit, Primary } = Controls;
+
+export const DeleteFilesControls = ({
+  onClose: _onClose,
+}: {
+  onClose?: () => void;
+}): React.JSX.Element => {
+  const {
+    disableCancel,
+    disableClose,
+    disablePrimary,
+    onClose,
+    onCancel,
+    onStart,
```

Comment:
Might be out of scope for this PR.

Since this is getting closer to "locking in" the view hook output - should we be firming these up? For example, I think the Primary/Cancel components are going to ActionStart/ActionCancel and it would be good to contextualize `onClose` in a similar manner?

Reply:
Should we revisit when that's been finalized or has that already been decided?

Reply:
Think we can do that in a follow up